src/runtime/meter-toast-api.js

```diff
@@ -29,12 +29,26 @@ import {warn} from '../utils/log';
 export const IFRAME_BOX_SHADOW =
   'rgba(60, 64, 67, 0.3) 0px -2px 5px, rgba(60, 64, 67, 0.15) 0px -5px 5px';
 export const MINIMIZED_IFRAME_SIZE = '420px';
+export const DEFAULT_IFRAME_URL = '/metertoastiframe';
+
+/**
+ * Properties:
+ * - iframeUrl: Relative URL of the iframe, e.g. "/meteriframe".
+ * - iframeUrlParams: List of extra params appended to the URL.
+ *
+ * @typedef {{
+ *   iframeUrl: (string|undefined),
+ *   iframeUrlParams: (?Object<string, string>|undefined),
+ * }}
+ */
+export let MeterToastApiParams;
 
 export class MeterToastApi {
   /**
    * @param {!./deps.DepsDef} deps
+   * @param {?MeterToastApiParams=} params
    */
-  constructor(deps) {
+  constructor(deps, params = undefined) {
```

Comment:
You can destructure function parameters with a default ([reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#setting_a_function_parameters_default_value))
```suggestion
  constructor(deps, { iframeUrl = DEFAULT_IFRAME_URL, iframeUrlParams = {} } = {}) {
```

Reply:
Thanks. The reason I didn't do this was because I wasn't sure how the jsdoc shoulds be. Please review if it is correct.